packages/design-system/src/components/Banner/Banner.js

```diff
@@ -13,6 +13,7 @@ const baseClass = 'banner';
 
 export const Banner = props => {
   const {
+    children,
```

Comment:
remove `text` property from component cause it's not needed anymore. Remember to update types in `banner.d.ts`

Reply:
done